docs/DeveloperGuide.adoc

```diff
@@ -294,18 +294,21 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
-|`* * *` |user |add a new person |
-
-|`* * *` |user |delete a person |remove entries that I no longer need
-
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
-
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
-
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|Priority |As a ... |I want to ... |So that ...
+|`* * *` |careless user |undo/redo any accidental deletion of memes |I can retrieve back my favourite memes
+|`* * *` |creative person |create my own memes from meme templates |
+|`* * *` |memey person |Easily share my memes to other platforms |I can send them in my chats
+|`* * *` |meme lover |manage my collection of memes |I can view them anytime I want
+|`* * *` |organised meme lover |tag memes |I can better organize my memes
+|`* * *` |user |get hints on command syntax when i type |I don't have to memorise the command syntax
+|`* * *` |user with many memes |filter memes |I can quickly locate a certain meme
+|`* * *` |user |import memes |I can import memes my friends shared with me into my personal collection
+|`* * *` |user |export memes |I can backup the files or share them with another user
+|`* * *` |user |archive memes |I can stop seeing old memes
+|`* * *` |user |view statistics of my meme usage |I know which memes are more liked/used
+|`* *` |bored person |view random memes |I can kill time
+|`* *` |user |bookmark my favourite meme |I can find my favourite memes quickly
+|`*` |person new to meme |get the reference to the meme I see for the first time |I can have fun as well
```

Comment:
Should we put the more 'basic' features on top (e.g. manage meme collection) of more 'advanced' features (e.g. undo/redo)?